distrobox-init

```diff
@@ -415,64 +415,111 @@ if [ "${upgrade}" -ne 0 ] ||
 		if ! apk add "${shell_pkg}"; then
 			shell_pkg="bash"
 		fi
-		deps="
-			${shell_pkg}
-			alpine-base
-			bash
-			bash-completion
-			bc
-			bzip2
-			coreutils
-			curl
-			diffutils
-			docs
-			findmnt
-			findutils
-			gcompat
-			gnupg
-			gpg
-			iproute2
-			iputils
-			keyutils
-			less
-			libc-utils
-			libcap
-			lsof
-			man-pages
-			mandoc
-			mount
-			musl-utils
-			ncurses
-			ncurses-terminfo
-			net-tools
-			openssh-client
-			pigz
-			pinentry
-			posix-libc-utils
-			procps
-			rsync
-			shadow
-			su-exec
-			sudo
-			tar
-			tcpdump
-			tree
-			tzdata
-			umount
-			unzip
-			util-linux
-			util-linux-misc
-			util-linux-login
-			vte3
-			wget
-			which
-			xauth
-			xz
-			zip
-			$(apk search -q mesa-dri)
-			$(apk search -q mesa-vulkan)
-			vulkan-loader
-		"
+		if apk add wolfi-base; then
```

Comment:
I would declare the common packages in `deps` outside of the "if" and then just add the packages of difference augmenting the common ones like:

```sh
deps="$deps
foo
bar
baz
"
```

Reply:
Cleaned up the deps lists :)